content/zh-cn/designs/telemetry_archiving/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "遥测存档"
+weight: 85
+draft: true
+---
+<!-- {{< synopsis-archiving >}} -->
+保存设备的测量值，并以原始或处理过的形式使用。
+<!--more-->
+
+## 挑战
+为了提供洞察力，IoT解决方案需要支持对其产生的信息进行实时、批处理和预测分析。在利用历史数据时，每种分析模式都能获得更好的信息。*同时*，未来还有目前未被理解的分析方法，因此IoT解决方案必须以尽可能灵活的方式存档数据，以满足未来的需求.
+
+## 解决方案
+IoT解决方案存储原始未处理过的传感器数据，并支持对这些原始样本进行有序的重放，从而确保业务可以获得最新且不断发展的洞察集合。存储和重放能力(store-and-replay)应使历史原始样本看起来几乎就像样本按正常时间序列到达一样。
+
+下图中显示的"遥测存档"设计可以提供此功能。
+
+![Telemetry Archiving Architecture](archiving.png) ([PPTx](atlas-archiving.pptx))
```

Comment:
This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

Reply:
The link is changed to  /designs/iot-atlas-patterns.pptx